v2/tools/generator/internal/astmodel/types.go

```diff
@@ -18,11 +18,13 @@ type TypeDefinitionSet map[TypeName]TypeDefinition
 // A restricted interface to indicate that the
 // consumer won’t modify the contained types.
 type ReadonlyTypes interface {
-	FullyResolve(t Type) (Type, error)
-	Get(t TypeName) TypeDefinition
-	TryGet(t TypeName) (TypeDefinition, bool)
+	Get(name TypeName) TypeDefinition
+	TryGet(name TypeName) (TypeDefinition, bool)
+	GetOrErr(name TypeName) (TypeDefinition, error)
```

Comment:
FWIW, I once worked on a team that had this very useful convention:

* Find() - method looked for something that might not be there, returned item or nil
* Get() - method looked for something that must be there, returning the item or throwing

Something like that might be better than the current method names.